packages/dxf-serializer/tests/testCsgToDxf.js

```diff
@@ -9,14 +9,14 @@ test('CSG to DXF 3DFACE', t => {
   const csg1 = new CSG()
   t.is(csg1.polygons.length,0)
 
-  const obs1 = serialize(csg1)
+  const obs1 = serialize({}, csg1)
   const exp1 = [empty]
   t.deepEqual(obs1,exp1)
 
   const csg2 = CSG.cube()
   t.is(csg2.polygons.length,6)
 
-  const obs2 = serialize(csg2)
+  const obs2 = serialize({}, csg2)
```

Comment:
hmm this (and the similar calls) makes me wonder if we should not detect the amount of params and default to ... defaults for options if there is only one parameter passed to serialize()

Reply:
Nah... It will be documented. Also, these functions would be used by developers so some understanding of parameter passing is expected. ;)